aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -18,25 +18,27 @@ use move_vm_runtime::{
     config::VMConfig, move_vm::MoveVM, native_extensions::NativeContextExtensions,
 };
 use std::ops::Deref;
+use aptos_types::on_chain_config::{FeatureFlag, Features};
 
 pub struct MoveVmExt {
     inner: MoveVM,
     chain_id: u8,
+    #[allow(unused)]
+    features: Features,
```

Comment:
Considering you don't need this, in this PR, rather than create an eyesore, let's introduce it in the PR where you need it.

We can call this the cleanup / plumbing PR.

Reply:
Fixed.